homeassistant/components/wemo.py

```diff
@@ -61,9 +61,8 @@ def stop_wemo(event):
 
     def discovery_dispatch(service, discovery_info):
         """Dispatcher for WeMo discovery events."""
-        # name, model, location, mac
-        model_name = discovery_info.get('model_name')
-        serial = discovery_info.get('serial')
+        model_name = discovery_info[1]
```

Comment:
Are you sure about that? I have updated all the netdisco output to be based on dictionaries.